### PR TITLE
Update serlio changelog

### PR DESCRIPTION
**Pull request by chr11115**
Update serlio changelog

### CODE REVIEW

File: doc/changelog.md [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
-* Added support for Maya 2020.
+## v1.2.0 (2022-07-XX)
+
+### Added:
+* Support for Maya 2020 and 2022. (#73)
+* About dialog with version number and license info.
+* Help tab with links to: 
+  * Serlio Website
+  * CGA Reference
+  * RPK Manual.
```
Comment (mistafunk) [inline doc/changelog.md:11]: move these lines to the bottom, not that important for users

File: doc/changelog.md [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+* Support for RGB-based opacity maps. (#15, #33, #18)
+* UX to warn user in case of:
+  * RPK/CGB and PRT version mismatch
+  * CGA generation errors
+  * Asset errors
+
+### Changed:
+* Updated Procedural Runtime (PRT) to 2.6 (corresponds to CityEngine 2021.1) and exclude non-relevant extensions.
+* Updated compilers (now using C++ 17)
+* Updated test framework and build system
```
Comment (mistafunk) [inline doc/changelog.md:24]: move to separate section "Development"

**Review by mistafunk — CHANGES_REQUESTED** [review]
Nice work, please sort the lists to what is important from an user point of view.

File: doc/changelog.md [hunk 3]
```diff
@@ -1,8 +1,46 @@
...
+* Only check out git repo once in Jenkins (#79)
+* Avoid unpacking of rule package. (performance improvement) (#74)
+* Improved rule attribute ordering. (matching CityEngine)
```
[the remark this answers is not in the document]
Comment (chr11115) [reply doc/changelog.md]: fixed.

File: doc/changelog.md (hunk 2 above)
Comment (chr11115) [reply doc/changelog.md]: done.

File: doc/changelog.md [hunk 4]
```diff
@@ -1,8 +1,46 @@
...
+* UX to warn user in case of:
+  * RPK/CGB and PRT version mismatch
+  * CGA generation errors
+  * Asset errors
+
+### Changed:
+* Updated Procedural Runtime (PRT) to 2.6 (corresponds to CityEngine 2021.1) and exclude non-relevant extensions.
+* Updated compilers (now using C++ 17)
+* Updated test framework and build system
+* Only check out git repo once in Jenkins (#79)
```
[the remark this answers is not in the document]
Comment (chr11115) [reply doc/changelog.md]: removed.

**Review by mistafunk — APPROVED** [review]
LGTM

**Pull request merged**